guides/source/testing.md

```diff
@@ -863,13 +863,16 @@ end
 
 If you want to use a remote browser, e.g.
 [Headless Chrome in Docker](https://github.com/SeleniumHQ/docker-selenium),
-you have to add remote `url` through `options`.
+you have to add remote `url`  and set `browser` as remote through `options`.
 
 ```ruby
 require "test_helper"
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  options = ENV["SELENIUM_REMOTE_URL"].present? ? { url: ENV["SELENIUM_REMOTE_URL"] } : {}
+  options = {
+    browser: ENV["SELENIUM_REMOTE_URL"].blank? ? :chrome : :remote,
+    url: ENV["SELENIUM_REMOTE_URL"].blank? ? nil : ENV["SELENIUM_REMOTE_URL"]
+  }
```

Comment:
I think it's clearer if we just use an if-statement:
```suggestion
  options = if ENV["SELENIUM_REMOTE_URL"].present?
    { browser: :remote, url: ENV["SELENIUM_REMOTE_URL"] }
  else
    { browser: :chrome } 
  end
```

Reply:
> I think it's clearer if we just use an if-statement:
> 
> ```suggestion
> 
>   options = if ENV["SELENIUM_REMOTE_URL"].present?
> 
>     { browser: :remote, url: ENV["SELENIUM_REMOTE_URL"] }
> 
>   else
> 
>     { browser: :chrome } 
> 
>   end
> 
> ```

Good idea. What abbaut using `present?`? We had cases, where the url was just `""` and the error message did not help to figure out this issue. I think it is safer to use `empty?` here.

Reply:
I think you can also use `ENV.fetch` with a default instead of multiple if/else:
https://docs.ruby-lang.org/en/master/ENV.html#method-c-fetch

Reply:
OK, together it would be:

```ruby
url = ENV.fetch("SELENIUM_REMOTE_URL", nil)
options = if url
    { browser: :remote,  url }
  else
    { browser: :chrome } 
  end
```

Can we create a new PR, as @tan-linx is not responding :)?